tests/test.py

```diff
@@ -1455,6 +1357,7 @@ def fork_translator(self, translator_kwargs, popen_kwargs=None):
 
         cli_args = ['--{}={}'.format(k, v) for k, v in default_config.items()]
 
+        print(cli_args)
```

Comment:
don't forget to remove it) or use `logger`/`logging` for that, because it configurable

Reply:
Sure:)